### PR TITLE
add EMEA end user discussion group time

### PR DESCRIPTION
**Pull request by sharrmander**
EMEA discussion group is now on the calendar :D

### CODE REVIEW

**Review by cartermp — APPROVED** [review]

File: content/en/blog/2022/new-end-user-resources.md [hunk 1]
```diff
@@ -25,8 +25,10 @@ observability powered by OpenTelemetry.
...
 - [Chatham House Rule](https://www.chathamhouse.org/about-us/chatham-house-rule)
   applies. Meetings will not be recorded.
-- The first meeting is on the [OpenTelemetry calendar](https://github.com/open-telemetry/community#calendar), scheduled for July 14th
-  2022 at 9AM PST. We intend to alternate months between APAC friendly and AMER
+- The first AMER friendly meeting is on the [OpenTelemetry calendar](https://github.com/open-telemetry/community#calendar), scheduled for July 14th
+  2022 at 9AM PST. This meeting series is scheduled to repeat every 8 weeks.
+- The first EMEA friendly meeting is on the [OpenTelemetry calendar](https://github.com/open-telemetry/community#calendar), scheduled for August 16th 2022 at 11AM CET. This meeting series is scheduled to repeat every 8 weeks.
+- We intend to alternate months between APAC friendly and AMER
   friendly timezones, we will re-evaluate this plan depending on attendance and
   feedback.
```
Comment (chalin) [inline content/en/blog/2022/new-end-user-resources.md:33]:
```suggestion
- We intend to alternate months between APAC- and AMER-friendly
  timezones. We will re-evaluate this plan based on attendance and
  feedback.
```

**Review by chalin — COMMENTED** [review]
LGTM. See inline comment for a suggested edit.

Also, some projects have a policy of not updating blog posts (other than copy edits or to fix links). Might we want to add something like a **last updated** notice towards the top?

> (**Last updated**: 2022-08-06) The [OpenTelemetry End User Working Group](https://github.com/open-telemetry/community/tree/main/working-groups/end-user) has heard feedback from ...

File: content/en/blog/2022/new-end-user-resources.md (hunk 1 above)
Comment (chalin) [reply content/en/blog/2022/new-end-user-resources.md]: @sharrmander - I"m going to assume that you're ok with the edits suggested above.

**Review by chalin — APPROVED** [review]
Thanks all!

**Comment by chalin** [comment]
Ok, let's get this merged!

**Pull request merged**